autonav_ws/src/autonav_odometry/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.8)
+project(autonav_odometry)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake_python REQUIRED)
+find_package(rclpy REQUIRED)
+find_package(autonav_shared REQUIRED)
```

Comment:
you might need a `find_package(autonav_msgs REQUIRED)` in here too, I think, can't remember